server/events/comment_parser.go

```diff
@@ -99,7 +99,7 @@ type CommentParseResult struct {
 // - atlantis plan -w staging -d dir --verbose
 // - atlantis plan --verbose -- -key=value -key2 value2
 //
-func (e *CommentParser) Parse(comment string, vcsHost models.VCSHostType) CommentParseResult {
+func (e *CommentParser) Parse(comment string, vcsHost models.VCSHostType, applyDisabled bool) CommentParseResult {
```

Comment:
Can `applyDisabled` be a field on the `CommentParser` struct? It doesn't need to be passed in as an argument because unlike comment and vcsHost it doesn't change based on the request, it's always the same once atlantis starts